tests/AddTasks.test.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { axe, toHaveNoViolations } from 'jest-axe';
+import AddTasks from '../app/components/AddTasks/AddTasks';
+
+// Mock Zustand stores
+jest.mock('@/lib/store/tag', () => ({
+  useTagStore: jest.fn(() => ({
+    fetchTags: jest.fn(),
+    tags: [
+      { id: '1', name: 'Tag 1' },
+      { id: '2', name: 'Tag 2' },
+      { id: '3', name: 'Tag 3' },
+    ],
+    setCurrentTag: jest.fn(),
+  })),
+}));
```

Comment:
These could probably be easily thrown in a reusable module so we can use them in other tests.